test/kickstart-templates/includes/post-system.cfg

```diff
@@ -20,3 +20,9 @@ EOF
 # user login, this results in more than requested amount of red-boot-induced reboots
 # and system needing much more time to roll back.
 ln -sf /dev/null /etc/systemd/user/grub-boot-success.timer
+
+# Force configuration for inotify watches, which are used when interacting with the
+# apiserver, as in getting logs, for example. These are the same defaults as OCP uses.
+echo "fs.inotify.max_user_watches = 65536" >> /etc/sysctl.conf
+echo "fs.inotify.max_user_instances = 8192" >> /etc/sysctl.conf
+sysctl --system
```

Comment:
What did we decide to do about these settings? Should we document them?

Reply:
Yes, I will open a ticket.